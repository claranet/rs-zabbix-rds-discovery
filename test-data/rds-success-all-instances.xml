<DescribeDBInstancesResponse xmlns="http://rds.amazonaws.com/doc/2014-10-31/">
  <DescribeDBInstancesResult>
    <DBInstances>
      <DBInstance>
        <AllocatedStorage>100</AllocatedStorage>
        <DBParameterGroups>
          <DBParameterGroup>
            <DBParameterGroupName>default.mysql5.6</DBParameterGroupName>
            <ParameterApplyStatus>in-sync</ParameterApplyStatus>
          </DBParameterGroup>
        </DBParameterGroups>
        <AvailabilityZone>us-east-1b</AvailabilityZone>
        <SecondaryAvailabilityZone>us-east-1a</SecondaryAvailabilityZone>
        <DBSecurityGroups/>
        <Iops>1000</Iops>
        <EnhancedMonitoringResourceArn>arn:aws:logs:us-east-1:1234567890:log-group:RDSOSMetrics:log-stream:db-IXRXA2XS7KFFA6JWYYWFZEBJDE</EnhancedMonitoringResourceArn>
        <EngineVersion>5.6.39</EngineVersion>
        <MasterUsername>mysqldbadmin</MasterUsername>
        <InstanceCreateTime>2018-03-28T19:54:07.871Z</InstanceCreateTime>
        <DBInstanceClass>db.m4.xlarge</DBInstanceClass>
        <ReadReplicaDBInstanceIdentifiers/>
        <MonitoringInterval>60</MonitoringInterval>
        <DBInstanceStatus>available</DBInstanceStatus>
        <BackupRetentionPeriod>7</BackupRetentionPeriod>
        <KmsKeyId>arn:aws:kms:us-east-1:1234567890:key/######################</KmsKeyId>
        <OptionGroupMemberships>
          <OptionGroupMembership>
            <OptionGroupName>default:mysql-5-6</OptionGroupName>
            <Status>in-sync</Status>
          </OptionGroupMembership>
        </OptionGroupMemberships>
        <LatestRestorableTime>2018-03-28T20:10:00Z</LatestRestorableTime>
        <CACertificateIdentifier>rds-ca-2015</CACertificateIdentifier>
        <DbInstancePort>0</DbInstancePort>
        <DbiResourceId>db-IXRXA2XS7KFFA6JWYYWFZEBJDE</DbiResourceId>
        <PreferredBackupWindow>05:27-05:57</PreferredBackupWindow>
        <DBInstanceIdentifier>mysqldb</DBInstanceIdentifier>
        <DBInstanceArn>arn:aws:rds:us-east-1:1234567890:db:mysqldb</DBInstanceArn>
        <Endpoint>
          <HostedZoneId>Z2R2ITUGPM61AM</HostedZoneId>
          <Address>mysqldb.########.us-east-1.rds.amazonaws.com</Address>
          <Port>3306</Port>
        </Endpoint>
        <Engine>mysql</Engine>
        <PubliclyAccessible>true</PubliclyAccessible>
        <IAMDatabaseAuthenticationEnabled>false</IAMDatabaseAuthenticationEnabled>
        <PerformanceInsightsEnabled>false</PerformanceInsightsEnabled>
        <MultiAZ>true</MultiAZ>
        <MonitoringRoleArn>arn:aws:iam::1234567890:role/rds-monitoring-role</MonitoringRoleArn>
        <DomainMemberships/>
        <StorageEncrypted>true</StorageEncrypted>
        <DBSubnetGroup>
          <VpcId>vpc-########</VpcId>
          <Subnets>
            <Subnet>
              <SubnetIdentifier>subnet-########</SubnetIdentifier>
              <SubnetStatus>Active</SubnetStatus>
              <SubnetAvailabilityZone>
                <Name>us-east-1e</Name>
              </SubnetAvailabilityZone>
            </Subnet>
            <Subnet>
              <SubnetIdentifier>subnet-########</SubnetIdentifier>
              <SubnetStatus>Active</SubnetStatus>
              <SubnetAvailabilityZone>
                <Name>us-east-1d</Name>
              </SubnetAvailabilityZone>
            </Subnet>
            <Subnet>
              <SubnetIdentifier>subnet-########</SubnetIdentifier>
              <SubnetStatus>Active</SubnetStatus>
              <SubnetAvailabilityZone>
                <Name>us-east-1c</Name>
              </SubnetAvailabilityZone>
            </Subnet>
            <Subnet>
              <SubnetIdentifier>subnet-########</SubnetIdentifier>
              <SubnetStatus>Active</SubnetStatus>
              <SubnetAvailabilityZone>
                <Name>us-east-1f</Name>
              </SubnetAvailabilityZone>
            </Subnet>
            <Subnet>
              <SubnetIdentifier>subnet-########</SubnetIdentifier>
              <SubnetStatus>Active</SubnetStatus>
              <SubnetAvailabilityZone>
                <Name>us-east-1a</Name>
              </SubnetAvailabilityZone>
            </Subnet>
            <Subnet>
              <SubnetIdentifier>subnet-########</SubnetIdentifier>
              <SubnetStatus>Active</SubnetStatus>
              <SubnetAvailabilityZone>
                <Name>us-east-1b</Name>
              </SubnetAvailabilityZone>
            </Subnet>
          </Subnets>
          <SubnetGroupStatus>Complete</SubnetGroupStatus>
          <DBSubnetGroupDescription>default</DBSubnetGroupDescription>
          <DBSubnetGroupName>default</DBSubnetGroupName>
        </DBSubnetGroup>
        <VpcSecurityGroups>
          <VpcSecurityGroupMembership>
            <VpcSecurityGroupId>sg-########</VpcSecurityGroupId>
            <Status>active</Status>
          </VpcSecurityGroupMembership>
        </VpcSecurityGroups>
        <LicenseModel>general-public-license</LicenseModel>
        <PendingModifiedValues/>
        <PreferredMaintenanceWindow>fri:05:57-fri:06:27</PreferredMaintenanceWindow>
        <StorageType>io1</StorageType>
        <AutoMinorVersionUpgrade>true</AutoMinorVersionUpgrade>
        <CopyTagsToSnapshot>false</CopyTagsToSnapshot>
      </DBInstance>
      <DBInstance>
        <AllocatedStorage>100</AllocatedStorage>
        <DBParameterGroups>
          <DBParameterGroup>
            <DBParameterGroupName>default.oracle-ee-12.1</DBParameterGroupName>
            <ParameterApplyStatus>in-sync</ParameterApplyStatus>
          </DBParameterGroup>
        </DBParameterGroups>
        <AvailabilityZone>us-east-1c</AvailabilityZone>
        <SecondaryAvailabilityZone>us-east-1f</SecondaryAvailabilityZone>
        <DBSecurityGroups/>
        <Iops>1000</Iops>
        <EnhancedMonitoringResourceArn>arn:aws:logs:us-east-1:1234567890:log-group:RDSOSMetrics:log-stream:db-YVS5NRBNHPGJZ3IT3WADXYSWYU</EnhancedMonitoringResourceArn>
        <EngineVersion>12.1.0.2.v11</EngineVersion>
        <MasterUsername>oracledbadmin</MasterUsername>
        <InstanceCreateTime>2018-03-28T20:00:48.832Z</InstanceCreateTime>
        <DBInstanceClass>db.m4.xlarge</DBInstanceClass>
        <ReadReplicaDBInstanceIdentifiers/>
        <MonitoringInterval>60</MonitoringInterval>
        <DBInstanceStatus>available</DBInstanceStatus>
        <BackupRetentionPeriod>7</BackupRetentionPeriod>
        <KmsKeyId>arn:aws:kms:us-east-1:1234567890:key/###################</KmsKeyId>
        <OptionGroupMemberships>
          <OptionGroupMembership>
            <OptionGroupName>default:oracle-ee-12-1</OptionGroupName>
            <Status>in-sync</Status>
          </OptionGroupMembership>
        </OptionGroupMemberships>
        <LatestRestorableTime>2018-03-28T20:14:30.818Z</LatestRestorableTime>
        <CACertificateIdentifier>rds-ca-2015</CACertificateIdentifier>
        <DbInstancePort>0</DbInstancePort>
        <DbiResourceId>db-YVS5NRBNHPGJZ3IT3WADXYSWYU</DbiResourceId>
        <PreferredBackupWindow>07:39-08:09</PreferredBackupWindow>
        <DBInstanceIdentifier>oracledb</DBInstanceIdentifier>
        <DBInstanceArn>arn:aws:rds:us-east-1:1234567890:db:oracledb</DBInstanceArn>
        <Endpoint>
          <HostedZoneId>Z2R2ITUGPM61AM</HostedZoneId>
          <Address>oracledb.########.us-east-1.rds.amazonaws.com</Address>
          <Port>1521</Port>
        </Endpoint>
        <Engine>oracle-ee</Engine>
        <PubliclyAccessible>true</PubliclyAccessible>
        <IAMDatabaseAuthenticationEnabled>false</IAMDatabaseAuthenticationEnabled>
        <PerformanceInsightsEnabled>false</PerformanceInsightsEnabled>
        <DBName>ORCL</DBName>
        <MultiAZ>true</MultiAZ>
        <CharacterSetName>AL32UTF8</CharacterSetName>
        <MonitoringRoleArn>arn:aws:iam::1234567890:role/rds-monitoring-role</MonitoringRoleArn>
        <DomainMemberships/>
        <StorageEncrypted>true</StorageEncrypted>
        <DBSubnetGroup>
          <VpcId>vpc-########</VpcId>
          <Subnets>
            <Subnet>
              <SubnetIdentifier>subnet-########</SubnetIdentifier>
              <SubnetStatus>Active</SubnetStatus>
              <SubnetAvailabilityZone>
                <Name>us-east-1e</Name>
              </SubnetAvailabilityZone>
            </Subnet>
            <Subnet>
              <SubnetIdentifier>subnet-########</SubnetIdentifier>
              <SubnetStatus>Active</SubnetStatus>
              <SubnetAvailabilityZone>
                <Name>us-east-1d</Name>
              </SubnetAvailabilityZone>
            </Subnet>
            <Subnet>
              <SubnetIdentifier>subnet-########</SubnetIdentifier>
              <SubnetStatus>Active</SubnetStatus>
              <SubnetAvailabilityZone>
                <Name>us-east-1c</Name>
              </SubnetAvailabilityZone>
            </Subnet>
            <Subnet>
              <SubnetIdentifier>subnet-########</SubnetIdentifier>
              <SubnetStatus>Active</SubnetStatus>
              <SubnetAvailabilityZone>
                <Name>us-east-1f</Name>
              </SubnetAvailabilityZone>
            </Subnet>
            <Subnet>
              <SubnetIdentifier>subnet-########</SubnetIdentifier>
              <SubnetStatus>Active</SubnetStatus>
              <SubnetAvailabilityZone>
                <Name>us-east-1a</Name>
              </SubnetAvailabilityZone>
            </Subnet>
            <Subnet>
              <SubnetIdentifier>subnet-########</SubnetIdentifier>
              <SubnetStatus>Active</SubnetStatus>
              <SubnetAvailabilityZone>
                <Name>us-east-1b</Name>
              </SubnetAvailabilityZone>
            </Subnet>
          </Subnets>
          <SubnetGroupStatus>Complete</SubnetGroupStatus>
          <DBSubnetGroupDescription>default</DBSubnetGroupDescription>
          <DBSubnetGroupName>default</DBSubnetGroupName>
        </DBSubnetGroup>
        <VpcSecurityGroups>
          <VpcSecurityGroupMembership>
            <VpcSecurityGroupId>sg-########</VpcSecurityGroupId>
            <Status>active</Status>
          </VpcSecurityGroupMembership>
        </VpcSecurityGroups>
        <LicenseModel>bring-your-own-license</LicenseModel>
        <PendingModifiedValues/>
        <PreferredMaintenanceWindow>tue:08:39-tue:09:09</PreferredMaintenanceWindow>
        <StorageType>io1</StorageType>
        <AutoMinorVersionUpgrade>true</AutoMinorVersionUpgrade>
        <CopyTagsToSnapshot>false</CopyTagsToSnapshot>
      </DBInstance>
    </DBInstances>
  </DescribeDBInstancesResult>
  <ResponseMetadata>
    <RequestId>298f362b-e14a-4ee0-9840-4546c276014a</RequestId>
  </ResponseMetadata>
</DescribeDBInstancesResponse>
