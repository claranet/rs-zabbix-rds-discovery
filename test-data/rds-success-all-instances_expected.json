{
  "data": [
    {
      "{#DB}": "mysqldb",
      "{#DB_ENDPOINT}": "mysqldb.########.us-east-1.rds.amazonaws.com",
      "{#DB_PORT}": 3306
    },
    {
      "{#DB}": "oracledb",
      "{#DB_ENDPOINT}": "oracledb.########.us-east-1.rds.amazonaws.com",
      "{#DB_PORT}": 1521
    }
  ]
}
